crates/dkg-cli/README.md

```diff
@@ -0,0 +1,146 @@
+# DKG CLI Tool
+
+Command-line tool for Distributed Key Generation (DKG) and key rotation protocols.
+
+### Fresh DKG Steps
+
+1. Initization
+
+a. Initialize a committee, assuming an admin and members. 
```

Comment:
let's use the name coordinator instead of admin? since it doesn't have any extra abilities